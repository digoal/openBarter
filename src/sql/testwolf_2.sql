
\i sql/model.sql
RESET client_min_messages;
RESET log_error_verbosity;
SET client_min_messages = notice;
SET log_error_verbosity = terse;

select * from fsubmitorder('a',NULL,'q2',10,NULL,'q1',20,NULL,NULL);
-- select * from fproducemvt();

select * from fsubmitorder('b',NULL,'q3',10,NULL,'q2',20,NULL,NULL);
-- select * from fproducemvt();
select * from fsubmitorder('c',NULL,'q1',10,NULL,'q3',20,NULL,NULL);
-- select * from fproducemvt();
select * from femptystack();
select id,nbc,nbt,grp,xid,xoid,own_src,own_dst,qtt,nat,ack from tmvt;

select * from acceptmvt();
select * from acceptmvt();
select * from acceptmvt();

select count(*) from tmvt;
