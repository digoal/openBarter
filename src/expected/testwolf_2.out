\i sql/model.sql
\set ECHO none
psql:sql/model.sql:3: NOTICE:  schema "test" does not exist, skipping
 _create_roles 
---------------
             0
(1 row)

 _grant_read 
-------------
 
(1 row)

 _grant_read 
-------------
 
(1 row)

 _grant_read 
-------------
 
(1 row)

 fifo_init 
-----------
 
(1 row)

 _grant_read 
-------------
 
(1 row)

 fifo_init 
-----------
 
(1 row)

psql:sql/model.sql:633: ERROR:  function deletemvt() does not exist
 id | diag 
----+------
  1 |    0
(1 row)

 id | diag 
----+------
  2 |    0
(1 row)

 id | diag 
----+------
  3 |    0
(1 row)

 femptystack 
-------------
           3
(1 row)

 id | nbc | nbt | grp | xid | xoid | own_src | own_dst | qtt | nat | ack 
----+-----+-----+-----+-----+------+---------+---------+-----+-----+-----
  1 |   3 |   3 |   1 |   3 |    3 | c       | b       |  20 | q3  | f
  2 |   3 |   3 |   1 |   2 |    2 | b       | a       |  20 | q2  | f
  3 |   3 |   3 |   1 |   1 |    1 | a       | c       |  20 | q1  | f
(3 rows)

 acceptmvt 
-----------
         1
(1 row)

 acceptmvt 
-----------
         1
(1 row)

 acceptmvt 
-----------
         1
(1 row)

 count 
-------
     0
(1 row)

